src/content/samples/node.mdx

```diff
@@ -25,11 +25,29 @@ $ git clone https://github.com/okteto/node-getting-started
 $ cd node-getting-started
 ```
 
-The `k8s.yml` file contains the [Kubernetes manifests](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) of the Node.js Sample App.
-Deploy a dev version of the application by executing:
+At the root of the directory, you'll find the `okteto.yaml` file. This describes how to [build](/docs/reference/manifest/#build-object-optional) and [deploy](/docs/reference/manifest/#deploy-string-optional) the Node Sample App.
+
+```
+build:
+  hello-world:
+    image: okteto.dev/node-hello-world:1.0.0
+    context: .
+deploy:
+  - kubectl apply -f k8s.yml
+dev:
+  hello-world:
+    image: okteto/dev:latest
+    command: bash
+    sync:
+      - .:/usr/src/app
+    forward:
+      - 9229:9229
+```
+
+Deploy your development environment by executing:
 
 ```console
-$ kubectl apply -f k8s.yml
+$ okteto deploy --build
 ```
 
 ```console
```

Comment:
update the console output:
https://github.com/okteto/docs/pull/119/files#diff-a7c7d0def22235aa498659f147beb3ee4800dfde172e7f81d3ab29244224de16R47

Reply:
all of it?

Reply:
What I usually do is follow the sample and add the output. In the golang sample, I used `...` for the build output, you can check it in this PR:
https://github.com/okteto/docs/pull/119